WooCommerce/Classes/Extensions/UserDefaults+Woo.swift

```diff
@@ -30,6 +30,10 @@ extension UserDefaults {
         case storePhoneNumber
         case hasSavedPrivacyBannerSettings
         case usedProductDescriptionAI
+
+        // Tooltip
+        case didUserDismissTooltip
```

Comment:
super nit: can probably be more consistent with other existing cases. also, do we plan to only have one use case for the tooltip? if we might have other use cases in the future, does it make sense to mention the current use case so that it doesn't sound like the user wants to dismiss all tooltips?

```suggestion
        case hasDismissedWriteWithAITooltip
```

Reply:
Done as part of https://github.com/woocommerce/woocommerce-ios/pull/10005/commits/b18551264d2497f159b4eb0dab4df66575a7d302